test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
- [ ] I can see that you added this file to` .gitignore` but it still got pushed to GitHub. This happened because you committed the file before adding it to gitignore.
This test.md file was not supposed to be pushed to github, therefore you need to untrack the file so that it will no longer be tracked by GitHub.

To untrack the file, run` git -rm --cached test. md`. This should help 👍🏽